### ISSUE TITLE
Symfony 7 Compatibility issues

### ISSUE THREAD

**Issue by uderline** [itsgoingd/clockwork#698]
Hello,

I'm using Laravel 11 (using Symfony 7).

During composer install, I get errors like:

```
Symfony\Component\ErrorHandler\Error\FatalError

Declaration of Clockwork\Support\Laravel\Console\CapturingFormatter::setDecorated(bool $decorated) must be compatible with Symfony\Component\Console\Formatter\OutputFormatterInterface::setDecorated(bool $decorated): void

  at vendor/itsgoingd/clockwork/Clockwork/Support/Laravel/Console/CapturingFormatter.php:27
     23▕
     24▕ 		return $capturedOutput;
     25▕ 	}
     26▕
  ➜  27▕ 	public function setDecorated(bool $decorated)
     28▕ 	{
     29▕ 		return $this->formatter->setDecorated($decorated);
     30▕ 	}
```

This is normal because the signature is:
```php
public function setDecorated(bool $decorated): void;
```

The fix seems quite easy but changing this will break compatibility for Sf 6 that uses a different contract.

I saw some issues about compatibility. Do you have any plans ?

Btw, thanks for this awesome project !

**Comment by itsgoingd** [comment]
Hey, I will try to fix this asap, thanks.

**Comment by itsgoingd** [comment]
Fixed in Clockwork 5.2.2.

**Comment by uderline** [comment]
Perfect ! Thanks ;)
